patches/common/chromium/ignore_rc_check.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/build/toolchain/win/tool_wrapper.py b/build/toolchain/win/tool_wrapper.py
+index a76e926a8681..c43839a01211 100644
+--- a/build/toolchain/win/tool_wrapper.py
++++ b/build/toolchain/win/tool_wrapper.py
+@@ -258,7 +258,10 @@ class WinTool(object):
+       if rc_exe_exit_code == 0:
+         import filecmp
+         # Strip "/fo" prefix.
+-        assert filecmp.cmp(rc_res_output[3:], rcpy_res_output[3:])
++        # ------
++        # Temporarily ignore compares         # 
++        # Nightly builds use very large version numbers that fail this check
++        # assert filecmp.cmp(rc_res_output[3:], rcpy_res_output[3:])
```

Comment:
Having seen several comments that say "Temporarily"... please add a `FIXME(ZacWalk)` to this

Reply:
Maybe to it would be better to add a comment in the `.patches.yaml`?

Reply:
@ckerr Done.